configure

```diff
@@ -1403,6 +1405,10 @@ Optional Packages:
                           Set the default ISA spec version, default to
                           20191213, available options: 2.2, 20190608, 20191213
   --with-sim=qemu         Sets the base RISC-V Simulator, defaults to qemu
+  --with-languages=       Sets the enabled languages for GNU toolchain,
```

Comment:
GCC rather than GNU?

Reply:
I think the term GNU toolchain is clear. If it turns out to be too confusing, we might adjust.